casper/binary/binary_view.py

```diff
@@ -0,0 +1,72 @@
+"""The blockchain view module extends a view for blockchain data structures """
+from casper.safety_oracles.clique_oracle import CliqueOracle
+from casper.abstract_view import AbstractView
+from casper.binary.bet import Bet
+import casper.binary.binary_estimator as estimator
+
+import random as r
+
+
+class BinaryView(AbstractView):
+    """A view class that also keeps track of a last_finalized_block and children"""
+    def __init__(self, messages=None):
+        super().__init__()
```

Comment:
Don't add messages here through super, but down in BlockchainView. We should pick one.

Reply:
I say do super.

You have to define `add_messages` in child, but the parent will always do the "if none, and add_message" call